packaging/arch/snap-mgmt.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# Overlord management of snapd for package manager actions.
+# Implements actions that would be invoked in %pre(un) actions for snapd.
+# Derived from the snapd.postrm scriptlet used in the Ubuntu packaging for
+# snapd.
+
+set -e
+
+SNAP_MOUNT_DIR="/var/lib/snapd/snap"
+
+show_help() {
+    exec cat <<'EOF'
+Usage: snap-mgmt.sh [OPTIONS]
+
+A simple script to cleanup snap installations.
```

Comment:
`Packaging utility to help manage snapd installation` perhaps?

Reply:
It's something I've wanted to do for a while, but the problem is that I can't fit snap-mgmt into the model that the current debian packaging follows. It does everything *after* it's all gone, while I do it *before* because I can leverage the package manager to do file cleanup.

I specifically made snap-mgmt to augment the package manager's ability to do things. If the package manager isn't capable of the same granularity of RPM, it's going to need its own variant.

And by the way @bboozzoo, openSUSE packaging *does* use it, it's just that the packaging in Git is out of sync with the OBS packaging.

Reply:
And also, there's literally no reason to duplicate it. You're not forbidden from installing from packaging/fedora just because you're using Arch.